.github/workflows/test-template.yml

```diff
@@ -253,7 +253,7 @@ jobs:
       fail-fast: false
       matrix:
         clientRuntime: ['node', 'wasm']
-        flavor: ['js_pg', 'js_neon', 'js_libsql', 'js_planetscale']
+        flavor: ['js_pg', 'js_neon', 'js_libsql', 'js_planetscale'] # 'js_d1'
```

Comment:
This will be for later when we add ci support, right?

Reply:
Yes! In the next PR, I already tried it but it would block the merge then